.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: Build macOS, Ubuntu, Windows
+
+on:
+  pull_request:
+
+  workflow_dispatch:
+
+jobs:
+  release:
+    name: Target ${{ matrix.target }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # May use in future. Do not perform duplicate builds
+          # - os: ubuntu-latest
+          #   profile: release+static
+          #   target: x86_64-unknown-linux-musl
+          #   needs_musl: true
+
+          # - os: ubuntu-latest
+          #   profile: release+static+aarch64
+          #   target: aarch64-unknown-linux-musl
+          #   needs_musl: true
+
+          # - os: macos-11
```

Comment:
macOS is not duplicate for CI, we only build macOS currently on tag pushes

Reply:
I'm not following something. Isn't that true for all of these?